src/libs/Violations/ViolationsUtils.ts

```diff
@@ -0,0 +1,64 @@
+import reject from 'lodash/reject';
+import Onyx from 'react-native-onyx';
+import ONYXKEYS from '@src/ONYXKEYS';
+import {PolicyCategories, PolicyTags, Transaction, TransactionViolation} from '@src/types/onyx';
+
+const ViolationsUtils = {
```

Comment:
Is this the new export pattern? Asking because I haven't seen one yet like this.

API.ts, for example, does this:

```
export {write, makeRequestWithSideEffects, read};
```

Reply:
That probably works better. It's been a while since I worked in a `prefer-default-exports` environment, so I'm still trying to figure things out.

Reply:
This will be fixed in ViolationUtils, which is upstream of this.